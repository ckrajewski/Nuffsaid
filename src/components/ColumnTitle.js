import React from 'react';
import { Typography } from '@material-ui/core';

/**
  * ColumnTitle component renders the tile for each column
  * Shows prioirty name (i.e. error or warning) as well as the number
  * of messages
*/
export default function ColumnTitle(props) {
  const { priorityLabel, priorityValue, messageCount } = props;
  return (
    <div>
      <div>
        <Typography variant="h6">
          {`${priorityLabel} Type ${priorityValue}`}
        </Typography>
      </div>
      <div>
        <Typography variant="body1" gutterBottom>
          {`Count ${messageCount}`}
        </Typography>
      </div>
    </div>
  );
}
