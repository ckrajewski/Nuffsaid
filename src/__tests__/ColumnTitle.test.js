import React from 'react';
import { shallow, mount, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ColumnTitle from '../components/ColumnTitle';

configure({ adapter: new Adapter() });

describe('ColumnTitle component', () => {
  test('renders', () => {
    const wrapper = shallow(<ColumnTitle />);
    expect(wrapper.exists()).toBe(true);
  });
});