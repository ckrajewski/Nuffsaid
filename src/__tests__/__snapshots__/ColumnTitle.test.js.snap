// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColumnTitle component renders 1`] = `
<div>
  <div>
    <h6
      className="MuiTypography-root MuiTypography-h6"
    >
      Error Type 1
    </h6>
  </div>
  <div>
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
    >
      Count 1
    </p>
  </div>
</div>
`;
